operator/Dockerfile

```diff
@@ -21,7 +21,10 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager
 
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
```

Comment:
It should also be possible to debug distroless images by using `debug` as tag or if you already use a tag by appending `-debug`. See https://github.com/GoogleContainerTools/distroless#debug-images

Reply:
Not for the `:nonroot` images 🙁
And images with root would not have been possible to run with the default security context 🙁